lib/libzfs/libzfs_sendrecv.c

```diff
@@ -3125,6 +3127,11 @@ zfs_receive_one(libzfs_handle_t *hdl, int infd, const char *tosnap,
 			VERIFY(0 == nvlist_add_uint64(props,
 			    zfs_prop_to_name(ZFS_PROP_CANMOUNT), 0));
 		}
+		if (0 == nvlist_lookup_nvlist(fs, "snapprops",
+			&lookup_nvlist)) {
```

Comment:
[style] Four space indent here for continued conditional.

```
+		if (0 == nvlist_lookup_nvlist(fs, "snapprops",
+		    &lookup_nvlist)) {
```

Or better yet if you picked a shorter variable name this could all be done on one line.  Since the scope of this variable is so small it doesn't need to be super descriptive.

```
                if (0 == nvlist_lookup_nvlist(fs, "snapprops", &lookup)) {
```